Dockerfile

```diff
@@ -1,14 +1,20 @@
 FROM ubuntu:20.04
 LABEL maintainer="Jeff Geerling"
 
+ENV DEBIAN_FRONTEND noninteractive
```

Comment:
This environment variable is also used for Debian 9/10. It also solves the problem of #2 or #3.